packages/frontend/src/components/profile/two_factor/TwoFactorAuth.js

```diff
@@ -108,16 +139,19 @@ const TwoFactorAuth = ({ twoFactor, history }) => {
                     twoFactorKind={twoFactor.kind}
                 />
             )}
-            {!twoFactor && (
+            {!twoFactor && isBrickedAccount && showBrickedAccountModal && (
```

Comment:
shouldn't it be `twoFactor` not `!twoFactor`? 
since it is 2fa enabled and we want to show modal upon disabling it?

Reply:
No, because for brickedaccounts, there is no value for twoFactor because we are unable to parse the account to get details such as the 2fa type and date of creation.

Reply:
Are you sure about this? I just created bricked account manually and I got this:

![image](https://user-images.githubusercontent.com/6027014/193352585-33fafaa3-7ce4-4798-b0f8-efb3ca56bb0b.png)

Reply:
When I click `Disable 2FA` button from banner it does show things correctly:
<img width="1155" alt="image" src="https://user-images.githubusercontent.com/6027014/193352911-f4c9718e-4217-4929-9619-c3936278417c.png">